main.tf

```diff
@@ -100,6 +105,24 @@ resource "aws_s3_bucket_public_access_block" "bucket" {
   restrict_public_buckets = true
 }
 
+# dynamodb table used to storing lock state
+resource "aws_dynamodb_table" "state_lock_table" {
+  count = var.dynamodb_state_locking ? 1 : 0
+
+  name           = "tf-state-lock-${var.application}"
+  billing_mode   = "PROVISIONED"
+  hash_key       = "LockID"
+  write_capacity = 1
+  read_capacity  = 1
```

Comment:
maybe up this to 5

Reply:
My opinion is that if you ever need more than the default 1, either: 

1.Make module variables for these options that default to 1
2. Set the table to use PAY_PER_REQUEST( aka On Demand) rather than PROVISIONED. 

It really depends on how often this is being used. I don't ever think of deployments as happening so frequently that scale becomes a big factor. If you set it provisioned, you pay for it even when deployments aren't happening.

Reply:
I think PAY_PER_REQUEST makes sense, deployments happen pretty infrequently. I could add 3 optional variables (billing_mode, write_capacity, read_capacity) but it might be overkill for this module.